test/test_show_address.py

```diff
@@ -0,0 +1,70 @@
+from ragger.navigator import NavInsID
```

Comment:
reject flows not tested for both usual and unusual derviation path
QR code display not tested

Reply:
Done.